This should be part of the Fedora empty-ruby_version patch

--- origsrc/ruby-2.0.0-p481/template/ruby.pc.in	2013-02-18 21:25:48.000000000 -0600
+++ src/ruby-2.0.0-p481/template/ruby.pc.in	2014-08-21 19:26:59.982177900 -0500
@@ -46,7 +46,7 @@ sitehdrdir=@sitehdrdir@
 
 Name: Ruby
 Description: Object Oriented Script Language
-Version: ${ruby_version}
+Version: @RUBY_PROGRAM_VERSION@
 URL: https://www.ruby-lang.org
 Cflags: -I${rubyhdrdir}/${arch} -I${rubyhdrdir}
 Libs: ${DLDFLAGS} ${LIBRUBYARG_SHARED} ${LIBS}
